core/store/src/rocksdb_metrics.rs

```diff
@@ -137,10 +147,10 @@ impl RocksDBMetrics {
     }
 }
 
-fn get_temperature_str(temperature: &near_store::Temperature) -> &'static str {
+fn get_temperature_str(temperature: &Temperature) -> &'static str {
     match temperature {
-        near_store::Temperature::Hot => "",
-        near_store::Temperature::Cold => "_cold",
+        Temperature::Hot => "_hot",
+        Temperature::Cold => "_cold",
```

Comment:
Add an entry to the changelog about a new metric and a change to the name of the old metric, please. So we don't forget to notify node maintainers (including us) about the need to change some dashboards.

Reply:
I'll remove _hot if there is a chance that it could break monitoring for validators. I would like the temperature to be a label but that's a discussion for another day.